src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
  * The `TemplateController` now accepts context argument
  * Deprecated *not* setting the "framework.router.utf8" configuration option as it will default to `true` in Symfony 6.0
  * Added tag `routing.expression_language_function` to define functions available in route conditions
+ * Added `assertThatForResponse` method to `BrowserKitAssertionsTrait` to throw an exception in test case on server exception.
```

Comment:
because `assertThatForResponse` might be too low level:
- Made `BrowserKitAssertionsTrait` report the original error message in case of a failure

Reply:
Done.